tests/test-definition.yaml

```diff
@@ -3,3 +3,6 @@ dimensions: []
 tests:
   - name: kerberos
     dimensions: []
+  # Requires manual connection to an AD cluster
+  - name: kerberos-ad
```

Comment:
Do you want to disable this test by default before merging?

Reply:
Yes, I think that makes sense to do.

Reply:
723898c4316d7188e457920395f2d8873470c911